Makefile

```diff
@@ -1,6 +1,23 @@
 SHELL := /bin/bash
 ROOT_DIR:= $(patsubst %/,%,$(dir $(realpath $(lastword $(MAKEFILE_LIST)))))
 
+GO_BUILD_OPTS := -mod=vendor
+GO_BUILD_TAGS := -tags "json1"
+
+GIT_COMMIT := $(or $(SOURCE_GIT_COMMIT),$(shell git rev-parse --short HEAD))
+OPM_VERSION := $(or $(SOURCE_GIT_TAG),$(shell git describe --always --tags HEAD))
```

Comment:
If we're building from release branches, we won't have tags to get the version from.

Reply:
Yeah, that's a good point, and I have that same thing tracked locally in my notes. I think I'd prefer if we instead shift towards attempting to have this information stored as a file in the repository somewhere as we currently do with that tracked file in the scripts (iirc?) directory. That way, we don't have to rely on portions of the root .git repository when building the container images, and instead read from those file(s) to get the registry/OLM version information.